### PR TITLE
feat(fe): add problem navigator

### LINKED ISSUE #237: [Feature] Problem - Navigator
### 🎨 Component Image

스크린샷 2022-09-28 오전 9 06 05

### ✏️ Describe component

problem 탭에 hambuger, refresh, run, submit, language를 선택할 수 있는 navigator

### ☁️ Props

현재 페이지가 보고 있는 문제의 이름

### 🔎 Using Page

problem

### 🏷 Required Component

_No response_

### PR DESCRIPTION
**Pull request by st42597**
### Description

스크린샷 2022-09-28 오전 9 17 39
problem 페이지의 navigator 구현

Close #237 

---

### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [ ] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: frontend/src/problem/components/Navigator.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    <div class="flex h-full w-full items-center justify-start gap-x-4">
+      <Dropdown class="pr-3">
+        <template #button>
+          <div class="flex h-9 w-fit flex-row items-center gap-x-2 text-white">
+            <Fa6SolidCaretDown class="h-4 w-4" />
+            <span>가파른 경사</span>
+          </div>
+        </template>
+        <template #items>
+          <ListItem>습격자 초라기</ListItem>
```
Comment (dotoleeoak) [inline frontend/src/problem/components/Navigator.vue:29]: `ListItem`이 import가 안 된 것 같아요~?

File: frontend/src/problem/components/Navigator.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import Button from '@/common/components/Atom/Button.vue'
+import Dropdown from '../../common/components/Molecule/Dropdown.vue'
+import IconDown from '~icons/fa6-solid/angle-down'
+import IconRefresh from '~icons/ep/refresh-right'
```
Comment (dotoleeoak) [inline frontend/src/problem/components/Navigator.vue:6]: 사소한거긴 한데, 가능하면 font awesome iconset 쓰는 게 전반적인 look & feel이 비슷해져서 좋을 것 같아요...!

```ts
import IconRefresh from '~icons/fa6-solid/arrows-rotate'
```

File: frontend/src/problem/components/Navigator.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+const activeItem = ref(items[0])
+
+const activeStyle = (item: string) => {
+  return item === activeItem.value ? `border-t-[3px] border-white pb-[3px]` : ''
+}
+</script>
+
+<template>
+  <div class="flex h-14 w-full flex-row items-center bg-slate-700 px-6">
```
Comment (dotoleeoak) [inline frontend/src/problem/components/Navigator.vue:19]: Context를 좀 더 살리기 위해서 `div` 대신 `nav` 쓰는 거 어떨까요?
(관련해서 "Semantic HTML" 검색해보세요)

**Review by dotoleeoak — CHANGES_REQUESTED** [review]
`Button` component가 main branch에 좀 변경사항이 있어서 main branch merge한 다음 확인 부탁드릴게요!

File: frontend/src/problem/components/Navigator.vue (hunk 2 above)
Comment (st42597) [reply frontend/src/problem/components/Navigator.vue:6]: font awesome에서 refresh 아이콘 찾아봤는데 두꺼운거밖에 없더라고요 ㅜㅜ

Comment (dotoleeoak) [reply frontend/src/problem/components/Navigator.vue:6]: 음... 좀 아쉽네요 그럼 일단 이대로 둘게요

**Review by dotoleeoak — APPROVED** [review]

**Pull request merged**